operator/gitops/argocd/pipeline-service/pipelines-as-code/bugfix.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pac-secret-manager
+  namespace: pipelines-as-code
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pac-secret-manager
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - list
+      - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: pac-secret-manager
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: pac-secret-manager
+subjects:
+  - kind: ServiceAccount
+    name: pac-secret-manager
+    namespace: pipelines-as-code
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: pac-secret-bugfix
+  namespace: pipelines-as-code
+spec:
+  concurrencyPolicy: Forbid
+  schedule: "*/10 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - image: quay.io/redhat-pipeline-service/ci-runner:main
+              command:
+                - /bin/bash
+                - -c
+                - |
+                  kubectl get secrets --all-namespaces -o json | \
+                    jq -r '.items[] |
+                      select(.metadata.name |
+                      startswith("pac-gitauth-")) |
+                      select(.metadata.creationTimestamp < (now - 86400)) |
+                      .metadata.namespace + " secret/" + .metadata.name
+                    ' | \
+                    xargs --max-args 2 --no-run-if-empty kubectl delete -n
```

Comment:
Is our ci-runner image produced from RHTAP production, or staging? I have a small concern that ProdSec might flag the image as not coming from a productized build system.

If possible, can we use registry.access.redhat.com/openshift4/ose-cli:v4.12? I _believe_ this image bundles `kubectl`, but if not you can use `oc` instead.

Reply:
Fixed. The image does contain `kubectl`.